packages/signature-controller/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.1.3]
+### Changed
+- Bump dependency on `@metamask/keyring-controller` to ^8.0.3
+- Bump dependency on `@metamask/logging-controller` to ^1.0.4
```

Comment:
Nit: It looks like this package is also a peer dependency, so we can be a bit more detailed here:

``` suggestion
- Bump dependency and peer dependency on `@metamask/logging-controller` to ^1.0.4
```